src/libworld/libworld.h

```diff
@@ -1870,6 +1875,7 @@ namespace world
         const string& airlineIcao() const { return m_airlineIcao; }
         const string& flightNo() const { return m_flightNo; }
         const string& callSign() const { return m_callSign; }
+        const string& currentAirportIcao() const { return ((m_phase == Phase::Arrival) ? m_plan->arrivalAirportIcao() : m_plan->departureAirportIcao()); }
```

Comment:
I think that in `currentAirportIcao()` we better return an empty string while en-route.

Reply:
You're right, I'll change that.